Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -4007,6 +4007,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         new DiscretePropertyWrapper<const AtomString&>(CSSPropertyHyphenateCharacter, &RenderStyle::hyphenationString, &RenderStyle::setHyphenationString),
         new DiscretePropertyWrapper<ImageOrientation>(CSSPropertyImageOrientation, &RenderStyle::imageOrientation, &RenderStyle::setImageOrientation),
         new DiscretePropertyWrapper<ImageRendering>(CSSPropertyImageRendering, &RenderStyle::imageRendering, &RenderStyle::setImageRendering),
+        new DiscretePropertyWrapper<DynamicRangeLimit>(CSSPropertyDynamicRangeLimit, &RenderStyle::dynamicRangeLimit, &RenderStyle::setDynamicRangeLimit),
```

Comment:
Looking at the spec, this is wrong, values should interpolate using `dynamic-range-limit-mix()`.

Reply:
Also, `RenderStyle::conservativelyCollectChangedAnimatableProperties` needs to take in account this, in order for `transition: all` to work properly with this CSS property.